pom.xml

```diff
@@ -94,7 +94,7 @@
     <test.args></test.args>
     <src.output>${basedir}/target</src.output>
     <imageio.ext.version>1.1.19</imageio.ext.version>
-    <jaiext.version>1.0.19</jaiext.version>
+    <jaiext.version>1.0-SNAPSHOT</jaiext.version>
```

Comment:
Do we need a new jai-ext version?

Reply:
Oh right... not for this fix, but yes, for gs-netcdf-out to work properly when using data packing, we do need a new jai-ext release indeed. It will come with a separate pull request.